### PR TITLE
Create trygveasp

### PR DESCRIPTION
**Pull request by trygveasp**

### CODE REVIEW

File: IcandoPRs/trygveasp [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (asmundb) [inline IcandoPRs/trygveasp]: Sure this will work?

**Review by asmundb — APPROVED** [review]
Better!

**Comment by trygveasp** [comment]
Everything was fine until @asmundb ruined everything:-(

**Comment by roelstappers** [comment]
Don't forget to create feature branches for PRs

**Pull request merged**